src/frontend/web_application/src/components/form/Switch/style.scss

```diff
@@ -16,5 +16,12 @@ $switch-height: 2rem !default;
 
   &__paddle {
     @include switch-paddle;
+    background-color: $co-color__fg__back--lower;
+  }
+
+  &__label {
+    padding-left: $co-grid__gutter--small;
+    font-size: .9rem;
```

Comment:
Does this might look weird in a form with multiple label etc. ?

Also may be we need to add font size modifiers to TextBlock, something like `<label><TextBlock inline small>bla</TextBlock></label>`. WDYT ?